src/components/AssetCard/AssetCard.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import { Network, Rarity, Item } from '@dcl/schemas'
+import { Mana } from '../Mana/Mana'
+import { Card } from '../Card/Card'
+import { Profile } from '../Profile/Profile'
+import './AssetCard.css'
+import { AssetImage } from '../AssetImage/AssetImage'
+
+export type AssetCardProps = {
+  asset: Pick<Item, 'id' | 'url' | 'name' | 'rarity' | 'network' | 'creator'>
+  action: React.ReactNode
+  actionIcon?: React.ReactNode
+  imagensrc: string
+  extraInformation: React.ReactNode
+  notForSale: boolean
+  price?: string
+  owners?: string
+  onClickCardURL: string
+}
+
+export const AssetCard = (props: AssetCardProps) => {
+  const {
+    action,
+    actionIcon,
+    asset,
+    extraInformation,
+    imagensrc,
+    notForSale,
+    onClickCardURL,
+    owners,
+    price
+  } = props
+
+  return (
+    <Card
+      className={'AssetCard'}
+      link
+      as={'a'}
+      to={`${onClickCardURL}${asset.url}`}
+      id={asset.id}
+    >
+      {asset ? (
+        <>
+          <AssetImage
+            name={asset.name}
+            rarity={asset.rarity}
+            src={imagensrc}
+            className={'AssetImage'}
+          />
+          <Card.Content data-testid="asset-card-content" className={'catalog'}>
+            <Card.Header>
+              <div className={'catalogTitle'}>
+                <span className={'textOverflow'}>{asset.name}</span>
+                {asset.network === Network.MATIC && (
+                  <span className={'creator'}>
+                    <Profile address={asset.creator} textOnly />
+                  </span>
+                )}
+              </div>
+            </Card.Header>
+            <div className={'CatalogItemInformation'}>
+              <span
+                className={`${'extraInformation'} ${
```

Comment:
You can use `classNames` to simplify the use of classes:
```suggestion
                className={classNames('extraInformation', notForSale && 'NotForSale')}
```